README.md

```diff
@@ -69,6 +69,10 @@ reporting the percentage of code covered. Start it with:
 npm run-script coverage
 ```
 
+### Docker
+
+The `docker-start` and `docker-test` scripts are deprecated as of version 2.8.2, and only remain for backwrads compatibility. Instead, developers should configure `blubber.yaml` and use [Blubber](https://github.com/wikimedia/blubber) to generate the desired Dockerfile.
```

Comment:
I'm thinking this should probably specify _service-runner_ 2.8.2.  That said: it looks like service-template-node depends on service-runner `^2.7.1` (i.e. strictly less than 2.8).  Would it make sense to also upgrade the service-runner dependency and just say "...scripts are deprecated and only remain..."?

Also: _backward_ is a typo

Reply:
> That said: it looks like service-template-node depends on service-runner ^2.7.1 (i.e. strictly less than 2.8).

Never mind, I see now that I misread that notation.  It still seems like it might be less confusing to omit the version entirely.

Reply:
Typo fixed, thanks for that catch! Version also removed.